access/email/Makefile

```diff
@@ -18,6 +18,7 @@ RELEASE_MESSAGE = "Building with GOOS=$(OS) GOARCH=$(ARCH)."
 
 DOCKER_NAME = teleport-plugin-email
 DOCKER_IMAGE = 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/$(DOCKER_NAME):$(VERSION)
+DOCKER_IMAGE_ECR_PUBLIC = public.ecr.aws/d7a2c0j1/$(DOCKER_NAME):$(VERSION)
```

Comment:
It'd be nice to loop back and fix `d7a2c0j1` once we have `gravitational`.  It would be confusing to have push to a different url than pull, even if they both work.

Reply:
Yes, I'll add a followup issue for this. I plan on replacing with `gravitational` once it is available.